doc/howtos/building-images.rst

```diff
@@ -230,17 +241,34 @@ The following instructions assume that swupd is not used.
 
 .. TODO: document how to create custom image recipes based on ostro-image.bbclass.
 
+.. _`ostro-image.bbclass`: https://github.com/ostroproject/meta-ostro/blob/master/meta-ostro/classes/ostro-image.bbclass
 
-Installing Additional Packages without swupd
---------------------------------------------
+Installing Additional Packages (without swupd)
+----------------------------------------------
+
+An image derived from ``ostro-image.bbclass`` without additional
+configuration is minimal and establishes a core OS with components
+that must always be present on a device. All additional components beyond
+this minimal configuration must be added explicitly by setting
+``OSTRO_IMAGE_EXTRA_FEATURES`` and/or ``OSTRO_IMAGE_EXTRA_INSTALL`` adding them
+to the image when building without swupd.  
```

Comment:
On Fri, 2016-04-29 at 10:44 -0700, David Kinder wrote:
> So should I drop the (without swupd) in the section title?

When doing that, the entire section needs to be rewritten in such a way
that it is clear which parts are general (adding to the base image) and
which parts are specific to swupd (adding to base image = extending
os-core bundle, defining additional bundles, adding to additional
bundles). The later part overlaps with what Igor wrote about swupd.

Considering the overlap, it's probably better to focus on the common
parts, just mention that "base image = os-core", and refer to the
additional document for additional swupd usage.